src/components/logo/z-logo/styles.css

```diff
@@ -1,18 +1,33 @@
 :host {
-  font-family: var(--dashboard-font);
-  font-weight: var(--font-rg);
+  display: inline-block;
+  vertical-align: top;
+  width: 31px;
+  height: 40px;
 }
 
-:host > div {
-  display: flex;
-  flex-direction: row;
-  justify-content: flex-start;
+img {
+  vertical-align: top;
+  content: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='31px' height='40px' viewBox='0 0 31 40'%3E%3Cg id='Symbols' stroke='none' stroke-width='1' fill='none' fill-rule='evenodd'%3E%3Cg id='Topbar/Mobile/Logged' transform='translate(-16.000000, -4.000000)'%3E%3Cg id='Group' transform='translate(16.000000, 4.000000)'%3E%3Cpolygon id='Fill-1' fill='%23E2001A' points='0 40 30.625 40 30.625 0 0 0'%3E%3C/polygon%3E%3Cpolygon id='Fill-2' fill='%23FEFEFE' points='8.17111587 6.20689655 23.4311708 6.20689655 23.4311708 11.671267 13.8691261 28.9163861 23.6647727 28.9163861 23.6647727 33.7931034 6.96022727 33.7931034 6.96022727 28.408218 16.522272 11.0831673 8.17111587 11.0831673'%3E%3C/polygon%3E%3C/g%3E%3C/g%3E%3C/g%3E%3C/svg%3E");
 }
 
-polygon {
-  fill: var(--myz-red);
+a {
+  display: inline-block;
+  width: 31px;
+  height: 40px;
```

Comment:
By giving to width and height the value of 100% of its parent you will not need to redefine them in the media query and you could also avoid the "style" tag for the link element in the index.tsx file

Reply:
👍🏻